docs/datamodel/props.rst

```diff
@@ -16,19 +16,19 @@ Every property is declared to have a specific
 :ref:`collection type <ref_datamodel_collection_types>` based on a scalar.
 
 There are two kinds of property item declarations: *abstract properties*,
-and *concrete properties*.  Abstract properties are defined on module level
+and *concrete properties*.  Abstract properties are defined on the module level
```

Comment:
We have a fairly strict linter limiting max line length to 79 characters. This line is a bit too long and causes the CI to fail.